R/Simulations-class.R

```diff
@@ -496,119 +496,108 @@ DualSimulations <- function(rho_est,
+# PseudoSimulations ----
 
-## -------------------------------------------------------------------------------
-## class for simulation using pseudo models
-## ------------------------------------------------------------------------
+## class ----
 
-##' Class `PseudoSimulations`
-##'
-##' This is a class which captures the trial simulations from designs using
-##' pseudo model. The design for DLE only responses and model from \code{\linkS4class{ModelTox}}
-##' class object. It contains all slots from \code{\linkS4class{GeneralSimulations}} object.
-##' Additional slots fit and stopReasons compared to the general class
-##' \code{\linkS4class{GeneralSimulations}}.
-##'
-##' @slot fit list of the final values. If samples are involved, these are the final fitted values.
-##' If no samples are involved, these are included the final modal estimates of the model parameters
-##' and the posterior estimates of the probabilities of the occurrence of a DLE.
-##' @slot FinalTDtargetDuringTrialEstimates the vector of all final estimates (the last estimate of) the TDtargetDuringTrial at the end
-##' of each simulations/when each trial stops
-##' @slot FinalTDtargetEndOfTrialEstimates vector of all final estimates or the last estimate of the TDtargetEndOfTrial when each trial
-##' stops
-##' @slot FinalTDtargetDuringTrialAtDoseGrid vector of the dose levels at dose grid closest below the final TDtargetDuringTrial estimates
-##' @slot FinalTDtargetEndOfTrialAtDoseGrid vector of  the dose levels at dose grid closest below the final TDtargetEndOfTrial estimates
-##' @slot FinalTDEOTCIs is the list of all 95% credibility interval of the final estimates of the TDtargetEndOfTrial
-##' @slot FinalTDEOTRatios is the vector of the ratios of the CI, the ratio of the upper to the lower 95% credibility intervals
-##' of the final estimates of the TDtargetEndOfTrial
-##' @slot FinalCIs list of all the final 95% credibility intervals of the TDtargetEndofTrial estimates or of the final optimal dose
-##' estimates when DLE and efficacy responses are incorporated after each simulations
-##' @slot FinalRatios vector of all the final ratios, the ratios of the upper to the lower 95% credibility interval of the
-##' final estimates of the TDtargetEndOfTrial or of the final optimal dose estimates (when DLE and efficacy responses are
-##' incorporated) after each simulations
-##' @slot stopReasons add slot description
-##' @slot stop_report matrix of stopping rule outcomes
-##'
-##' @export
+#' `PseudoSimulations`
+#'
+#' @description `r lifecycle::badge("stable")`
+#' This class captures trial simulations from designs using pseudo model.
+#' It has additional slots `fit` and `stop_reasons` compared to the
+#' general class [`GeneralSimulations`].
+#'
+#' @slot fit (`list`)\cr final fit values.
+#' @slot final_td_target_during_trial_estimates (`numeric`)\cr final estimates of the TDtargetDuringTrial.
```

Comment:
```suggestion
#' @slot final_td_target_during_trial_estimates (`numeric`)\cr final estimates of the `TDtargetDuringTrial`.
```
please 1) use backticks around these code variable names, and 2) consider the need for making them snake_case, I don't see right away if they refer to old names or not

Reply:
The slot is called "final_td_target_during_trial_estimates", not "td_target_during_trial" and there are many internal variables in different programs named as "TDtargetDuringTrial". I thought that could also mean something else. That is why I did not make the change in the description.